doc/source/changes/version_0_27.rst.inc

```diff
@@ -61,6 +61,42 @@ Miscellaneous improvements
 
 * added support for coloring numeric values for object arrays (e.g. arrays containing both strings and numbers).
 
+* allowed to sort values of an array along an axis (closes :issue:`225`):
```

Comment:
move this up in the list of misc improvements. This is probably one of the most interesting improvements